controllers/keda/util/util.go

```diff
@@ -70,11 +70,22 @@ func UpdateKedaControllerStatus(ctx context.Context, cl client.Client, kedaContr
 
 func RunningOnOpenshift(ctx context.Context, logger logr.Logger, cl client.Client) bool {
 	gvk := schema.GroupVersionKind{Group: "route.openshift.io", Version: "v1", Kind: "route"}
+	return isGvkPresent(ctx, logger, cl, gvk)
+}
+
+// RunningWithMonitoringStack returns true if the monitoring stack (i.e. Prometheus) is present, false otherwise
+func RunningWithMonitoringStack(ctx context.Context, logger logr.Logger, cl client.Client) bool {
+	gvk := schema.GroupVersionKind{Group: "monitoring.coreos.com", Version: "v1", Kind: "Prometheus"}
```

Comment:
Is this `gvk` criteria sufficient to detect the presence of the Prometheus in the cluster?

Reply:
I would prefer to skip this question and instead just check for presence of CRDs (see other comment)